SimplyE/Colors.xcassets/deleteAction.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
is the `Colors.xcasset` just for dark mode? or is it meant to include all colors?

Reply:
It includes colors for both light mode and dark mode. Since we don't have a light mode color palette right now, so the colors might not be very accurate. I have talked to Marcus and he says he is going to have a light mode color palette when he does the redesign.